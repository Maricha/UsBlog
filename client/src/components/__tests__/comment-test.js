import React from 'react';
import renderer from 'react-test-renderer';
import Comment from '../comment';

describe('Comment Component', () => {
  it('renders correctly', () => {
    const comment = {
      authorName: "test",
      content: "conent"
    }
    const tree = renderer.create(
      <Comment comment={comment} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  })
});