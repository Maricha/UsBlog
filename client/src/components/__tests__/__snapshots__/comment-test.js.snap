// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comment Component renders correctly 1`] = `
<div
  className="MuiGrid-container-5 MuiGrid-direction-xs-column-8 Component-commentWrapper-4"
>
  <div
    className="MuiGrid-item-6 MuiGrid-grid-xs-12-45 MuiGrid-grid-md-true-61"
  >
    <div
      className="Component-commentAuthor-2"
    >
      test
    </div>
  </div>
  <div
    className="MuiGrid-item-6 MuiGrid-grid-xs-12-45 MuiGrid-grid-md-true-61"
  >
    <div
      className="Component-commentContent-3"
    >
      conent
    </div>
  </div>
</div>
`;
